hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/client/MirrorNodeClient.java

```diff
@@ -269,11 +268,21 @@ public Schedule getScheduleInfo(String scheduleId) {
         return callRestEndpoint("/schedules/{scheduleId}", Schedule.class, scheduleId);
     }
 
+    public TokenBalancesResponse getTokenBalances(String tokenId) {
+        log.debug("Verify token balances '{}' is returned by Mirror Node", tokenId);
+        return callRestEndpoint("/tokens/{tokenId}/balances", TokenBalancesResponse.class, tokenId);
+    }
+
     public TokenInfo getTokenInfo(String tokenId) {
         log.debug("Verify token '{}' is returned by Mirror Node", tokenId);
         return callRestEndpoint("/tokens/{tokenId}", TokenInfo.class, tokenId);
     }
 
+    public TokensResponse getTokensResponse(String tokenId) {
```

Comment:
nit, keep naming consistent

```suggestion
    public TokensResponse getTokens(String tokenId) {
```

Reply:
Thanks I have updated it